LeaderboardBackend.Test/Runs.cs

```diff
@@ -65,21 +68,93 @@ public async Task SetUp()
             _categoryId = createdCategory.Id;
         }
 
+        [Test]
+        public static async Task GetRun_OK()
+        {
+            Run created = await CreateRun(
+                new()
+                {
+                    CategoryId = _categoryId,
+                    Info = "",
+                    PlayedOn = LocalDate.MinIsoValue,
+                    TimeOrScore = 1000,
+                    UserId = TestInitCommonFields.Admin.Id,
+                }
+            );
+
+            await FluentActions.Awaiting(() => _apiClient.Get<TimedRunViewModel>(
+                $"/api/run/{created.Id.ToUrlSafeBase64String()}",
+                new() { }
+            ))
+                .Should()
+                .NotThrowAsync()
+                .WithResult(new()
+                {
+                    Id = created.Id,
+                    Info = created.Info,
+                    PlayedOn = created.PlayedOn,
+                    CreatedAt = created.CreatedAt,
+                    UpdatedAt = created.UpdatedAt,
+                    DeletedAt = created.DeletedAt,
+                    CategoryId = created.CategoryId,
+                    UserId = created.UserId,
+                    Time = created.Time,
+                });
+        }
+
+        [Test]
+        public static async Task GetRun_NotFound_BadRouteParamType() =>
+            await FluentActions.Awaiting(() =>
+                _apiClient.Get<RunViewModel>(
+                "/api/run/1",
+                new() { }
+            )).Should()
+            .ThrowAsync<RequestFailureException>()
+            .Where(e => e.Response.StatusCode == HttpStatusCode.NotFound);
+
+        [Test]
+        public static async Task GetRun_NotFound_RunDoesNotExist() =>
+            await FluentActions.Awaiting(() =>
+                _apiClient.Get<RunViewModel>(
+                $"/api/run/{Guid.Empty.ToUrlSafeBase64String()}",
+                new() { }
+            )).Should()
+            .ThrowAsync<RequestFailureException>()
+            .Where(e => e.Response.StatusCode == HttpStatusCode.NotFound);
```

Comment:
These could be one test.

Reply:
I can't do that because the `TestCase` attribute doesn't like dynamic values, which `Guid.Empty.ToUrl...` is.

Reply:
I was thinking just stringify it manually. It becomes `"AAAAAAAAAAAAAAAAAAAAAA"`.